src/ripple/basics/Log.h

```diff
@@ -236,6 +235,25 @@ deprecatedLogs()
     return logs;
 }
 
+class LogSquelcher
+{
+public:
+    LogSquelcher()
+        : severity_(deprecatedLogs().severity())
+    {
+        deprecatedLogs().severity(
+            beast::Journal::Severity::kNone);
+    }
+
+    ~LogSquelcher()
+    {
+        deprecatedLogs().severity(severity_);
+    }
+
+private:
+    beast::Journal::Severity const severity_;
+};
```

Comment:
This is pretty cool. I think it would be a bit better if a `Logs&` was injected in the ctor and stored, instead of calling the global `deprecatedLogs()`.

Obviously, the callers will then have to use the global, but at least we're not introducing it here.